src/AbstractFixer.php

```diff
@@ -63,6 +63,10 @@ public function __construct()
 
     final public function fix(\SplFileInfo $file, Tokens $tokens)
     {
+        if ($this instanceof ConfigurableFixerInterface && null === $this->configuration) {
```

Comment:
this only test top level, i.e. options of a configuration can be required as well, shouldn't we test those as well?